affiliatewp-leaderboard.php

```diff
@@ -53,6 +53,24 @@ class AffiliateWP_LB_Requirements_Check extends AffiliateWP_Requirements_Check {
 	 */
 	protected $slug = 'affiliatewp-leaderboard';
 
+	/**
+	 * Add-on requirements.
+	 *
+	 * @since 1.0.0
```

Comment:
Should be `@since 1.1`
```suggestion
	 * @since 1.1
```